Vladimir Bykov's portfolio

vladjnbykov opened the pull request:
Site is on Netlify:
https://www.vladjnbykov.com/

vladjnbykov committed:
main structure, hero image, logos

vladjnbykov committed:
featured projects, other projects

vladjnbykov committed:
mobile view fixed

vladjnbykov committed:
fixing larger screens

vladjnbykov committed:
adding font awesome

vladjnbykov committed:
fixing code structure, accessibility, TAB-navigation

vladjnbykov committed:
readme file

KarolCislowski commented on index.html:
```diff
@@ -0,0 +1,415 @@
+
+<body>
+  <!-- HERO IMAGE-->
+  <header class="hero">
```
Your hero section is too tall. it should take about 40% of the height on the desktop.

KarolCislowski commented on index.html:
```diff
@@ -0,0 +1,415 @@
+      <h2 class="hero-subtitle1">Data scientist &</h2>
+      <h2 class="hero-subtitle2">front-end developer</h2>
+      <div>
+        <img class="my-photo" src="./images/my_photo1.jpg" alt="my photo"/>
```
The image should be slightly moved down to break the Hero section bottom line.

KarolCislowski commented on index.html:
```diff
@@ -0,0 +1,415 @@
+        <a href="https://github.com/vladjnbykov/ML-Breast-cancer-predictions-mRNA-microarray">
+          <picture class="pic-container">
+              <img src="./images/breast_cancer.png" alt="breast cancer PCA analysis"/>
+              <div class="project-img-title">BREAST CANCER PROGNOSIS</div>
```
this header is not clearly visible in the background of the photo 😿

KarolCislowski commented on index.html:
```diff
@@ -0,0 +1,415 @@
+            <span>Pandas</span>
+            <span>Seaborn</span>
+            <span>Matplotlib</span>
+            <span>Scikit-learn</span>
```
You should make your font size bigger because it is nearly impossible to read it. 

Give your tags some padding to make them look nicer :)

KarolCislowski commented on style.css:
```diff
@@ -0,0 +1,438 @@
+    background-color: rgb(234, 234, 250);
+    color: navy;
+}
+
```
provide some max-width to your page, because it doesn't look good on wide screens.

KarolCislowski commented on index.html:
```diff
@@ -0,0 +1,415 @@
+    </section>  
+  </section>
+
+<!-- OTHER PROJECTS-->
```
I think you need to re-think your font face here because it feels like one of the science journals from 80'/90'. I love it because it brings nice memories, but HR people will not understand it 🥼

KarolCislowski reviewed (changes requested):
Your page works nice, but it is a little bit too far from the design provided to you by us. 

Please fix the issues I pointed in the comments and it will be a very nice portfolio to show to the recruiters.

vladjnbykov committed:
response to comments

vladjnbykov committed:
removing unused images

vladjnbykov commented:
Hej Karol,

I believe I have now fixed the issues with larger screen, photo-picture 
which breaks out of the bottom line at least on the screens I tested, 
resolved issue with title on the first featured project by reducing 
contrast of the image, span-elements for skills I had to convert to divs 
in order to accomodate them on two lines with outlined background and 
paddings.
I did not fully understand your last comment, but I have changed now 
image from the chat-bot which we created in a team, so now several fonts 
are in that image and I also blured picture a bit.
Everything is now pushed on Github.

Best regards,
Vlad

------ Originalmeddelande ------
Från: "Karol Cislowski" ***@***.***>
Till: "Technigo/portfolio" ***@***.***>
Kopia: "Vladimir JN Bykov" ***@***.***>; "Author" 
***@***.***>
Skickat: 2021-03-10 10:27:52
Ämne: Re: [Technigo/portfolio] Vladimir Bykov's portfolio (#8)

>
>@KarolCislowski requested changes on this pull request.
>
>Your page works nice, but it is a little bit too far from the design 
>provided to you by us.
>
>Please fix the issues I pointed in the comments and it will be a very 
>nice portfolio to show to the recruiters.
>
>--------------------------------------------------------------------------------
>In index.html 
><https://github.com/Technigo/portfolio/pull/8#discussion_r591251193>:
>
> > +<head>
>
>+    <meta charset="UTF-8">
>
>+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
>
>+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
>
>+    <link rel="stylesheet" href="./style.css" />
>
>+    <link rel="preconnect" href="https://fonts.gstatic.com">
>
>+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
>
>+    <link rel="preconnect" href="https://fonts.gstatic.com">
>
>+    <link ***@***.***&display=swap" rel="stylesheet">
>
>+    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab%7CRuda" rel="stylesheet">
>
>+    <title>Vlad's Portfolio</title>
>
>+</head>
>
>+
>
>+<body>
>
>+  <!-- HERO IMAGE-->
>
>+  <header class="hero">
>
>
>Your hero section is too tall. it should take about 40% of the height 
>on the desktop.
>
>--------------------------------------------------------------------------------
>In index.html 
><https://github.com/Technigo/portfolio/pull/8#discussion_r591252113>:
>
> > +    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
>
>+    <link rel="preconnect" href="https://fonts.gstatic.com">
>
>+    <link ***@***.***&display=swap" rel="stylesheet">
>
>+    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab%7CRuda" rel="stylesheet">
>
>+    <title>Vlad's Portfolio</title>
>
>+</head>
>
>+
>
>+<body>
>
>+  <!-- HERO IMAGE-->
>
>+  <header class="hero">
>
>+    <picture>
>
>+      <h1 class="hero-title"><span>PORTFOLIO:</span> VLADIMIR BYKOV, MD PhD</h1>
>
>+      <h2 class="hero-subtitle1">Data scientist &</h2>
>
>+      <h2 class="hero-subtitle2">front-end developer</h2>
>
>+      <div>
>
>+        <img class="my-photo" src="./images/my_photo1.jpg" alt="my photo"/>
>
>
>The image should be slightly moved down to break the Hero section 
>bottom line.
>
>--------------------------------------------------------------------------------
>In index.html 
><https://github.com/Technigo/portfolio/pull/8#discussion_r591253544>:
>
> > +      pair-programming, Git, GitHub
>
>+    </p>
>
>+  </section>
>
>+
>
>+  <!--FEATURED PROJECTS-->
>
>+  <section class="featured-projects">
>
>+    <h3 class="outline-title" tabindex="0" role="featured projects">
>
>+      <span>FEATURED PROJECTS</span>
>
>+    </h3>
>
>+
>
>+    <section class="featured-projects-container">
>
>+      <article>
>
>+        <a href="https://github.com/vladjnbykov/ML-Breast-cancer-predictions-mRNA-microarray">
>
>+          <picture class="pic-container">
>
>+              <img src="./images/breast_cancer.png" alt="breast cancer PCA analysis"/>
>
>+              <div class="project-img-title">BREAST CANCER PROGNOSIS</div>
>
>
>this header is not clearly visible in the background of the photo 😿
>
>--------------------------------------------------------------------------------
>In index.html 
><https://github.com/Technigo/portfolio/pull/8#discussion_r591254758>:
>
> > +            <span>Python</span>
>
>+            <span>Pandas</span>
>
>+            <span>Seaborn</span>
>
>+            <span>Matplotlib</span>
>
>+            <span>Scikit-learn</span>
>
>
>You should make your font size bigger because it is nearly impossible 
>to read it.
>
>Give your tags some padding to make them look nicer :)
>
>--------------------------------------------------------------------------------
>In style.css 
><https://github.com/Technigo/portfolio/pull/8#discussion_r591256530>:
>
> > @@ -0,0 +1,438 @@
>
>+body {
>
>+    font-family: "Montserrat", sans-serif;
>
>+    margin: 0;
>
>+    background-color: rgb(234, 234, 250);
>
>+    color: navy;
>
>+}
>
>+
>
>
>provide some max-width to your page, because it doesn't look good on 
>wide screens.
>
>--------------------------------------------------------------------------------
>In index.html 
><https://github.com/Technigo/portfolio/pull/8#discussion_r591262972>:
>
> > +          </picture>
>
>+          <h4>CHATBOT: RESTAURANT FOOD ORDER</h4>
>
>+          <p>
>
>+            Web app for order food. Built with JavaScript.
>
>+          </p>
>
>+          <h6>
>
>+            <span>HTML5</span>
>
>+            <span>CSS3</span>
>
>+            <span>Javascript ES6</span>
>
>+          </h6>
>
>+        </a>
>
>+      </article>
>
>+    </section>
>
>+  </section>
>
>+
>
>+<!-- OTHER PROJECTS-->
>
>
>I think you need to re-think your font face here because it feels like 
>one of the science journals from 80'/90'. I love it because it brings 
>nice memories, but HR people will not understand it 🥼
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly, view it on GitHub 
><https://github.com/Technigo/portfolio/pull/8#pullrequestreview-608484305>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AQFHVNS7WLEVHKJCVEJTEWLTC43RRANCNFSM4YV5X3XA>.
>

KarolCislowski commented:
I think you need a container to align everything to the centre. You need to make your hero smaller too
![image](https://user-images.githubusercontent.com/50926159/111038089-79622c00-8427-11eb-9bf1-8302bb75c023.png)

vladjnbykov committed:
fixing central alignment for the whole page on large screens, hero image max width

vladjnbykov committed:
fixing font size on the large screen

vladjnbykov commented:
Oh, yes, right.
I fixed it by margin-auto and by max width on the hero image.
Have a nice weekend.
Best regards,
Vlad


------ Originalmeddelande ------
Från: "Karol Cislowski" ***@***.***>
Till: "Technigo/portfolio" ***@***.***>
Kopia: "Vladimir JN Bykov" ***@***.***>; "Author" 
***@***.***>
Skickat: 2021-03-13 18:12:59
Ämne: Re: [Technigo/portfolio] Vladimir Bykov's portfolio (#8)

>
>I think you need a container to align everything to the centre. You 
>need to make your hero smaller too
>image
><https://user-images.githubusercontent.com/50926159/111038089-79622c00-8427-11eb-9bf1-8302bb75c023.png>
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly, view it on GitHub 
><https://github.com/Technigo/portfolio/pull/8#issuecomment-798647003>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AQFHVNQCGTZJBKAVWWXV7DDTDOMJXANCNFSM4YV5X3XA>.
>

KarolCislowski reviewed (approved):
Have a great weekend:)
